lowstepper_wasm_demo/src/styles.css

```diff
@@ -1,28 +1,261 @@
-html, body {
-  margin: 0;
-  padding: 0;
-}
-
-.App {
-
-  font-family: sans-serif;
-  text-align: center;
-}
-
-p {
-  font-size: 16px;
-}
-
-.container {
-  width: 440px;
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  align-items: center;
-  margin: 0 auto;
-  margin-top:20px;
-}
-.container > * {
-  padding: 20px;
-
-}
+html, body {
+  margin: 0;
+  padding: 0;
+}
+
+.App {
+
+  font-family: sans-serif;
+  text-align: center;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  width: 440px;

```

Comment:
for some reason this looks enormous on my screen currently... not sure why, maybe 2x monitor?